src/components/pages/home/events.tsx

```diff
@@ -184,41 +181,43 @@ const Events = () => {
             )}
           </div>
 
-          <h3
-            className={`mb-2 mt-3 text-xl font-medium text-black transition-all duration-300`}
-            style={{
-              height: `${isMonthly ? monthlyCardHeight : upcomingCardHeight}px`,
-              overflow: 'hidden'
-            }}
-            title={title}
-          >
-            {title}
-          </h3>
+          <a href={link} target='_blank' rel='noopener noreferrer' className='block'>
```

Comment:
This is genius @iamnoufal 
Thanks a lot for your contribution. Though its a bit late to pull the PR, it is worth the wait. 
Works like a charm. 
Merging it right away. 

Cheers :)